featurebyte/routes/registry.py

```diff
@@ -271,13 +296,33 @@
 
 app_container_config.register_class(UseCaseService)
 app_container_config.register_class(UseCaseController)
+app_container_config.register_class(TargetTableTask)
+app_container_config.register_class(RandomTask)
+app_container_config.register_class(FeatureJobSettingAnalysisTask)
+app_container_config.register_class(FeatureJobSettingAnalysisBacktestTask)
+app_container_config.register_class(HistoricalFeatureTableTask)
+app_container_config.register_class(ObservationTableTask)
+app_container_config.register_class(ObservationTableUploadTask)
+app_container_config.register_class(DeploymentCreateUpdateTask)
+app_container_config.register_class(BatchRequestTableTask)
+app_container_config.register_class(BatchFeatureTableTask)
+app_container_config.register_class(MaterializedTableDeleteTask)
+app_container_config.register_class(BatchFeatureCreateTask)
+app_container_config.register_class(FeatureListCreateWithBatchFeatureCreationTask)
+app_container_config.register_class(StaticSourceTableTask)
+app_container_config.register_class(TileTask)
+app_container_config.register_class(OnlineStoreCleanupTask)
+app_container_config.register_class(LongRunningTask)
+app_container_config.register_class(TestTask)
```

Comment:
Do we need to register `RandomTask`?

Would be good to put all these tasks under a specific session (maybe add a comment to indicate we should register all tasks here).

Reply:
agree it's a little confusing at the moment, but at the moment it really depends on how it's used in the test, and how the task gets registered into the task registry as well.

i'll fix in a follow-up (will probably add a comment, or consolidate the registering properly) together with the other comment you had!